lib/network/protocol.js

```diff
@@ -337,7 +337,7 @@ Protocol.prototype.handleMirror = function(params, callback) {
       if (item.hash !== utils.rmd160(hasher256.digest())) {
         self._logger.warn('mirror read integrity check failed, destroying');
         report.end(ExchangeReport.FAILURE, 'FAILED_INTEGRITY');
-        item.shard.destroy();
+        item.shard.destroy(utils.noop);
```

Comment:
Should we log warning if there was an error on the [callback](https://github.com/Storj/kfs/blob/master/lib/write-stream.js#L63)?